packages/@aws-cdk/aws-elasticsearch/test/integ.elasticsearch.ts

```diff
@@ -0,0 +1,35 @@
+import { EbsDeviceVolumeType } from '@aws-cdk/aws-ec2';
+import { App, Construct, Stack, StackProps } from '@aws-cdk/core';
+import * as es from '../lib';
+
+class TestStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    new es.Domain(this, 'Domain', {
+      elasticsearchVersion: es.ElasticsearchVersion.ES_VERSION_7_1,
+      clusterConfig: {
+        masterNodes: 3,
+        masterNodeInstanceType: 'm4.large.elasticsearch',
```

Comment:
```suggestion
        masterNodeInstanceType: 'm5.large.elasticsearch',
```

Might as well use the latest instance types?